genesyscloud/telephony_provider_edges_trunkbasesettings/resource_genesyscloud_telephony_providers_edges_trunkbasesettings.go

```diff
@@ -109,32 +40,41 @@ func createTrunkBaseSettings(ctx context.Context, d *schema.ResourceData, meta i
 		Properties:    properties,
 	}
 
+	fmt.Printf("create inboundsite string in createTrunkBaseSettings: %s\n", inboundSiteString)
+	fmt.Printf("create site string in createTrunkBaseSettings: %s\n", siteString)
```

Comment:
Leftover debug statements? 

If we're keeping them we should use `log` instead of `fmt`

Reply:
Done